api/bazel/repositories.bzl

```diff
@@ -1,311 +1,22 @@
-BAZEL_SKYLIB_RELEASE = "0.5.0"
-BAZEL_SKYLIB_SHA256 = "b5f6abe419da897b7901f90cbab08af958b97a8f3575b0d3dd062ac7ce78541f"
-
-GOGOPROTO_RELEASE = "1.1.1"
-GOGOPROTO_SHA256 = "9f8c2ad49849ab063cd9fef67e77d49606640044227ecf7f3617ea2c92ef147c"
-
-PGV_RELEASE = "0.0.11"
-PGV_SHA256 = "d92c7f22929f495cf9f7d825c44f9190eda1d8256af321e3e8692570181b28a6"
-
-GOOGLEAPIS_GIT_SHA = "d642131a6e6582fc226caf9893cb7fe7885b3411"  # May 23, 2018
-GOOGLEAPIS_SHA = "16f5b2e8bf1e747a32f9a62e211f8f33c94645492e9bbd72458061d9a9de1f63"
-
-PROMETHEUS_GIT_SHA = "99fa1f4be8e564e8a6b613da7fa6f46c9edafc6c"  # Nov 17, 2017
-PROMETHEUS_SHA = "783bdaf8ee0464b35ec0c8704871e1e72afa0005c3f3587f65d9d6694bf3911b"
-
-OPENCENSUS_GIT_SHA = "ab82e5fdec8267dc2a726544b10af97675970847"  # May 23, 2018
-OPENCENSUS_SHA = "1950f844d9f338ba731897a9bb526f9074c0487b3f274ce2ec3b4feaf0bef7e2"
-
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@envoy//bazel:envoy_repository.bzl", "envoy_repository")
```

Comment:
This will break `data-plane-api` repo to be built alone. Rules under `api/` shouldn't rely on anything outside it.

Reply:
good catch - I didn't know about the data-plane repo. Let me make some changes